app/controllers/admin_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class AdminController < ApplicationController
   skip_before_action :check_date
+  skip_before_action :require_authenticated_user, only: %i[login validate]
```

Comment:
Doesn't this allow both admin and submitter access to these methods?
Do we want something similar but maybe just for admins here? :require_admin_user

Reply:
If your question was about if we should be allowing "skip_before_action", we have to skip it in order to not be continually redirected away from the sign-in page.  

But... Yikes!!!  I totally missed that you can log in as a submitter and then manually go to the /citations page, which you shouldn't be able to do!  

You're right.  The admin-only pages should be restricted to admins only.  I'll fix this on Monday.  Thanks for the catch!!

Reply:
This inspired me to put a number of new checks for access rights on the Manual Testing script in the Wiki!  Thanks!

Reply:
A related issue to this is the person with submitter id 3 shouldn't be able to view or edit the submitter profile for id 2, nor should they be able to see 2's submissions.

Reply:
Fixed